### ISSUE TITLE
Bisect_ppx not working well with other attributes

### ISSUE THREAD

**Issue by smondet** [rleonid/bisect_ppx#4]
```
File "src/pure/ketrew_path.mli", line 23, characters 0-28:
Unrecognized [@@deriving] annotation syntax
Command exited with code 2.
```

**Comment by smondet** [comment]
It seems that `bisect_ppx` is instrumenting the payload of attributes:

By removing the `ppx_deriving_yojson` call + option `-dsource`:

```ocaml
ocamlfind ocamlc -c -g -annot -bin-annot -package bisect_ppx -package cohttp -package docout -package nonstd -package ppx_blob  -package pvem -package sosa -package uri -package yojson -I src/pure -o src/pure/ketrew_path.cmi src/pure/ketrew_path.mli -dsource

type t[@@deriving
        let _ = Bisect.Runtime.init "src/pure/ketrew_path.mli"
        let _ = Bisect.Runtime.mark "src/pure/ketrew_path.mli" 0; yojson]
val file : string -> t
val directory : string -> t
val root : t
...
```

**Comment by rleonid** [comment]
I think that PR #8 fixes this:
```
$ ocamlfind ocamlc -c -g -annot -bin-annot -package bisect_ppx -package cohttp -package docout -package nonstd -package ppx_blob  -package pvem -package sosa -package uri -package yojson -I src/pure -o src/pure/ketrew_path.cmi src/pure/ketrew_path.mli -dsource
findlib: [WARNING] Interface topdirs.cmi occurs in several directories: /Users/leonidrozenberg/.opam/4.02.1/lib/ocaml/compiler-libs, /Users/leonidrozenberg/.opam/4.02.1/lib/ocaml
type t[@@deriving yojson]
val file : string -> t
val directory : string -> t
val root : t
val absolute_file_exn : string -> t
val absolute_directory_exn : string -> t
val relative_directory_exn : string -> t
val relative_file_exn : string -> t
val concat : t -> t -> t
val to_string : t -> string
val to_string_quoted : t -> string
val exists_shell_condition : t -> string
val size_shell_command : t -> string
```

**Comment by rleonid** [comment]
Closed by #8.
